datafusion/expr/src/expr.rs

```diff
@@ -697,13 +694,13 @@ impl AggregateFunction {
     }
 }
 
-/// WindowFunction
+/// A function used as a SQL window function
+///
+/// In SQL, you can use:
+/// - Actual window functions ([`WindowUDF`])
+/// - Noraml aggregate functions ([`AggregateUDF`])
```

Comment:
typo here: "Normal"

Reply:
Fixed in [6b1a243](https://github.com/apache/datafusion/pull/13393/commits/6b1a2439d44464d7f49c0c9f4df4f5f67f172344)